package.json

```diff
@@ -7,6 +7,7 @@
     "@lingui/detect-locale": "^3.13.0",
     "@lingui/react": "^3.13.0",
     "@reduxjs/toolkit": "^1.6.2",
+    "@sushiswap/sdk": "^5.0.0-canary.116",
```

Comment:
@aeolianeth are we sure we need to be using a canary build?

Reply:
@peripheralist that's a great catch, I was meaning to check if there was a stable version.

Looks like there's a `5.0.0`. so I'll give it a whirl. https://www.npmjs.com/package/@sushiswap/sdk/v/5.0.0

Reply:
@peripheralist Ok, very strange... the `5.0.0` doesn't have any code in it 🙃 

`https://registry.yarnpkg.com/@sushiswap/sdk/-/sdk-5.0.0.tgz#7e33337eef7adba5a13716aba0bcdcb6a48bae22`

| 5.0.0 | 5.0.0-canary.116 |
| --- | --- |
| <img width="184" alt="Screen Shot 2022-01-20 at 2 56 30 pm" src="https://user-images.githubusercontent.com/94939382/150279020-83750703-4376-43f3-a99d-01f322816b5b.png"> | <img width="166" alt="Screen Shot 2022-01-20 at 2 56 38 pm" src="https://user-images.githubusercontent.com/94939382/150279038-490a156d-1b29-4acb-bca1-e6a5f2de9587.png"> |

The latest non-canary release was [almost a year ago](https://www.npmjs.com/package/@sushiswap/sdk/v/4.1.1) (`v4.1.1`), which also makes me a bit hesitant. Upon trying it, it's not quite compatible with the changes in this PR (breaking changes, makes sense), but given there's no docs specific to this version (or any version, really), it might take some time to re-implement with `v4.1.1`.

I've fixed the version at `5.0.0-canary.116` for now. Curiously, the `https://github.com/sushiswap/sushiswap-sdk` repo is archived now (must have only happened in the last few days, unless I missed it originally) which is... also strange.

I'd be happy to re-implement with `v4.1.1` if you feel strongly. Let me know 😊

Reply:
strange... looking at their version history seems like they have a lot of canary versions, and those seem to have the most downloads. so maybe just odd versioning practice, or the sushi team just never does any testing and doesn't wanna be held accountable lol.

since it seems unlikely that bugs here would cause big issues in the app, let's just roll with the canary build for now

Reply:
@peripheralist  yeah lol the version history is quite confusing... I reached out in the Sushi Discord but nothing has come from that yet. Will keep an eye on it and make changes if necessary 👍